store/tikv/coprocessor.go

```diff
@@ -388,9 +387,9 @@ func (it *copIterator) run(ctx goctx.Context) {
 	}()
 }
 
-func (it *copIterator) sendToTaskCh(ctx goctx.Context, t *copTask) (finished bool, canceled bool) {
+func (it *copIterator) sendToTaskCh(ctx goctx.Context, t *copTask, taskCh chan<- *copTask) (finished bool, canceled bool) {
```

Comment:
Why change the arg list?

Reply:
I move taskCh out of `copIterator`, so the extra data is passed as argument.